app/src/main/java/io/homeassistant/companion/android/notifications/MessagingManager.kt

```diff
@@ -571,10 +594,38 @@ class MessagingManager @Inject constructor(
                     bluetoothAdapter?.enable()
             }
             COMMAND_BLE_TRANSMITTER -> {
+                val sensorDao = AppDatabase.getInstance(context).sensorDao()
                 if (title == TURN_OFF)
                     BluetoothSensorManager.enableDisableBLETransmitter(context, false)
                 if (title == TURN_ON)
                     BluetoothSensorManager.enableDisableBLETransmitter(context, true)
+                if (title == BLE_SET_ADVERTISE_MODE || title == BLE_SET_TRANSMIT_POWER)
+                    sensorDao.updateSettingValue(
+                        BluetoothSensorManager.bleTransmitter.id,
+                        if (title == BLE_SET_ADVERTISE_MODE)
+                            BluetoothSensorManager.SETTING_BLE_ADVERTISE_MODE
+                        else
+                            BluetoothSensorManager.SETTING_BLE_TRANSMIT_POWER,
+                        when (title) {
+                            BLE_SET_ADVERTISE_MODE -> {
+                                when (data[BLE_ADVERTISE]) {
+                                    BLE_ADVERTISE_BALANCED -> "balanced"
+                                    BLE_ADVERTISE_LOW_LATENCY -> "lowLatency"
+                                    BLE_ADVERTISE_LOW_POWER -> "lowPower"
```

Comment:
These should also be constants that are used in both BluetoothSensorManager.kt and here rather than just strings.